pulseguardian/static/js/users_listing.js

```diff
@@ -5,4 +5,38 @@
 $(document).ready(function() {
     // DataTables
     $('#pulse_users').dataTable();
+    var usersTable = $('#users').DataTable();
+
+    $('#users tbody').on('click', '.admin-check', function(){
+    	$('#userAlert').addClass('hide');
+
+    	var userData = usersTable.row($(this).closest('tr')).data();
+    	var confirmMsg = 'Change the admin role to ' + userData[0] + ' ?';
+    	var confirmation = confirm(confirmMsg);
+
+    	if(confirmation){
+    		var isAdmin = $(this).is(':checked');
+    		var urlApi = '/user/' + $(this).data('uid') + '/set-admin'
+
+    		$.ajax({
+    			type: 'PUT',
```

Comment:
Actually sorry, I was thinking in Python terms here. This is probably fine. I don't have a validator file but that's an excellent idea.